src/projects.html

```diff
@@ -100,20 +72,12 @@ <h2 class="section-heading">Jaspals Voice</h2>
             <div class="col-xs-12 col-md-6 project">
                 <h2 class="section-heading">Pimp My Cause</h2>
                 <h3>Django, Python, HTML, CSS, JavaScript, Heroku</h3>
-                <p>Pimp My Cause connects marketers, researchers, and
-                innovators who wish to volunteer to good causes in need of
-                innovative solutions. The organisation has successfully
-                matchmade over 1,300 charities and volunteers since its
-                inception in 2011.</p>
-                <p>WHFNP volunteers are tackling a website rebuild project
-                for this charity to modernise the technology stack and
-                improve the site's user experience. Using Django, Sass and
-                JavaScript, the end goal is to leave Pimp My Cause with a
-                slick new website that is robust, sustainable, and capable
-                of supporting the growth in its membership expected over
-                the next few years.</p><a class="button project-info"
-                href="https://github.com/womenhackfornonprofits/pimp-my-cause-django">View
-                Pimp My Cause Project</a>
+                <p>Pimp My Cause connects marketers, researchers, and innovators who wish to volunteer to good causes in need of innovative solutions. The organisation has successfully matchmade over 1,300 charities and volunteers since its inception in
+                    2011.</p>
+                <p>WHFNP volunteers are tackling a website rebuild project for this charity to modernise the technology stack and improve the site's user experience. Using Django, Sass and JavaScript, the end goal is to leave Pimp My Cause with a slick new
+                    website that is robust, sustainable, and capable of supporting the growth in its membership expected over the next few years.</p>
+                <a class="button project-info" href="https://github.com/womenhackfornonprofits/pimp-my-cause-django">View Open Source Project</a>
+                <a class="button project-info" href="https://staging-pimpmycause.herokuapp.com/">Visit website</a>
```

Comment:
yes please let's use standard button only and remove the extra code and class. Also if a button doesn't have a ink just delete the html for now. No need to hide with css if thats what you mean.